lib/components/collapse.vue

```diff
@@ -43,10 +43,9 @@
         },
 
         watch: {
-            visible(newVal, oldVal){
-                if (newVal !== this.show) {
-                    this.toggle()
-                }
+            visible(newVal) {
+                this.show = newVal;
```

Comment:
Why removed condition guard? (newVal !== oldVal) this helps prevent emitting duplicate events